frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx

```diff
@@ -129,10 +129,14 @@ export const OperatorHubPage = withFallback(
           <PageHeading title="OperatorHub" />
           <p className="co-catalog-page__description">
             Discover Operators from the Kubernetes community and Red Hat partners, curated by Red
-            Hat. Operators can be installed on your clusters to provide optional add-ons and shared
-            services to your developers. Once installed, the capabilities provided by the Operator
-            appear in the <a href="/catalog">Developer Catalog</a>, providing a self-service
-            experience.
+            Hat. You can purchase commercial software through{' '}
+            <ExternalLink
+              href="https://marketplace.redhat.com/en-us?utm_source=openshift_console"
+              text="Red Hat Marketplace"
+            />
+            . You can install Operators on your clusters to provide optional add-ons and shared
+            services to your developers. After installation, the Operator capabilities will appear
+            in the <a href="/catalog">Developer Catalog</a> providing a self-service experience.
```

Comment:
This should use `Link`, not an `a` element

Reply:
Pushed change in amended commit.